src/milia/api/async_export.cljs

```diff
@@ -74,12 +76,14 @@
 
 (defn build-export-suffix
   "Build the export options string to pass to the Ona API."
-  [data-format export-options]
-  (->> export-options
-    ((apply juxt export-option-values))
-    (map add-param export-option-keys)
-    (concat ["export_async.json?format=" data-format])
-    (apply str)))
+  ([url export-options]
+   (build-export-suffix url nil export-options))
+  ([url data-format export-options]
```

Comment:
The order of the arts shouldn't change as you add more, ie `export-options` should stay the second arg.

You can set the last arg to be optional and default nil with this:

```clojure
[url export-options & [data-format]]
```

Simpler and more concise than the two signatures.